Project/Source Files/Screens/SearchTab/SearchViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SearchTabViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
+
+	required init?(coder aDecoder: NSCoder) { fatalError("init(coder:) has not been implemented") }
+	init(searchController: UISearchController = UISearchController(searchResultsController: nil)) {
+		self.searchController = searchController
+		super.init(nibName: nil, bundle: nil)
+	}
+	private var customView: SearchView {
+		return view as! SearchView
+	}
+	private var searchController: UISearchController
+	private var searchScopeButtonTitles = ["Title", "Genre"]
+	override func loadView() {
+		view = SearchView.instanceFromNib()
+	}
+    override func viewDidLoad() {
+        super.viewDidLoad()
+		setupTableView()
+		setupVC()
+		setupSearchController()
+    }
+	private func setupTableView() {
+		customView.tableView.register(
+			UINib(nibName: "SearchViewCell", bundle: nil),
+			forCellReuseIdentifier: SearchViewCell.searchViewCellReuseIdentifier)
+		customView.tableView.delegate = self
+		customView.tableView.dataSource = self
+		customView.tableView.rowHeight = 150
+	}
+	private func setupSearchController() {
+		navigationItem.searchController = searchController
+		searchController.searchBar.placeholder = NSLocalizedString("Search", comment: "")
+		navigationItem.hidesSearchBarWhenScrolling = false
+		searchController.hidesNavigationBarDuringPresentation = false
+		searchController.obscuresBackgroundDuringPresentation = false
+		searchController.searchBar.delegate = self
+		searchController.searchBar.scopeButtonTitles = searchScopeButtonTitles
+	}
+	private func setupVC() {
```

Comment:
generally functions shouldn't have abbreviations. It;s better to write more code, but be more descriptive.